### DISCUSSION TITLE
[Feature] use webdav instead of local storage

### DISCUSSION THREAD

**Discussion by IamSAL** [immich-app/immich#13822] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

i m currently hosting immich, but i have very limited on site storage, not enough to keep all my pics and videos as i would like.

i also happen to have access to unlimited webdav storage, and i m looking for a service that would function like immich but instead of storing the library on local disk, it should store it on the webdav directory that i specify ( i need it to not consume the local storage).

i considered using rclone or simple webdav synchronization but it's too crude and i need more than basic synching , i need gallery functionality.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1687
